packages/ffe-accordion-react/src/AccordionItem.js

```diff
@@ -11,77 +11,53 @@ const createClasses = (baseClass, isOpen, type) =>
     });
 
 class AccordionItem extends Component {
-    constructor(props) {
-        super();
-        this.state = {
-            isOpen: props.isOpen || false,
-        };
+    state = {
+        isOpen: this.props.isOpen || false,
```

Comment:
`this.state.isOpen` is never read if `this.props.isOpen` is set, so this can just default to `false` in all cases.